h/accounts/test/views_test.py

```diff
@@ -79,9 +80,14 @@ def mock_flash_function():
                                 return_value=None)
 
 
-@pytest.mark.usefixtures('routes_mapper')
 class TestAuthController(object):
 
+    @pytest.fixture(autouse=True)
+    def routes(self, config):
+        config.add_route('forgot_password', '/forgot')
+        config.add_route('index', '/index')
+        config.add_route('stream', '/stream')
```

Comment:
P.S. When I've been defining fixtures and helper methods in classes I've been putting them at the bottom of the class (which seemed to match the style I'd seen elsewhere in our code of putting fixtures and helpers at the bottom of modules.